### PR TITLE
Add Jerom's documentations

### PR DESCRIPTION
**Pull request by SarahAlidoost**
- This PR moves this [documentation](https://hpc-to-cluster-docs.readthedocs.io/en/latest/#) to the main readthedocs. Please see the source [here](https://github.com/jeromaerts/hpc_to_cluster_docs/blob/master/docs/source/index.rst).
- It fixes factual errors, but not any other improvements

### CODE REVIEW

File: docs/hpc_to_cluster.rst [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+-  ``docker://ewatercycle/{model}-grpc4bmi:{version}``
+
+**5. Adjust code to run Singularity container**
+
+Code should be adjusted to run Singularity instead of Docker following:
+::
+
+    from grpc4bmi.bmi_client_singularity import BmiClientSingularity
+
+    model = BmiClientSingularity(image='ewatercycle-wflow-grpc4bmi.sif', input_dir=input_dir, output_dir=output_dir)
```
Comment (sverhoeven) [inline docs/hpc_to_cluster.rst]: Can we use the new syntax?
```suggestion
    model = BmiClientSingularity(image='ewatercycle-wflow-grpc4bmi.sif', input_dirs=[input_dir], work_dir=work_dir)
```

File: docs/hpc_to_cluster.rst [hunk 2]
```diff
@@ -0,0 +1,266 @@
+Migrate from HPC to Cluster (Cartesius) guide
+=============================================
+Familiarize yourself with Linux by reading this simple guide:
```
Comment (sverhoeven) [inline docs/hpc_to_cluster.rst:5]: Can we explain why migration is needed?
```suggestion
The HPC node jupyter.ewatercycle.org can be used for small test experiments, to do actual work you will need to run your notebook/script on the cluster (Cartesius). On Cartesius the forcing data is already present and many users can run jobs at the same time without interfering each other.

Familiarize yourself with Linux by reading this simple guide:
```

File: docs/hpc_to_cluster.rst [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+
+- https://maker.pro/linux/tutorial/basic-linux-commands-for-beginners
+
+*************************
+Migratation Preparation
```
Comment (sverhoeven) [inline docs/hpc_to_cluster.rst]:
```suggestion
Migration Preparation
```

File: docs/hpc_to_cluster.rst [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+``/home/{YourUsernameOnTheCartesius}/``
+
+Cleanup files in the scratch directory.
+
+
+**************************************
+Submitting Jupyter Job on Cluster node
+**************************************
+Here we briefly explain general SBATCH parameters and how to launch a Jupyter Lab environment on Cartesius. Start by opening a text editor on Cartesius (e.g. ``vim``) or (easier) your local machine (e.g. notepad). Copy the following text inside your text editor, edit the Conda environment name, and save as **run_jupyter_on_cartesius.sh** (make sure the extension is ``.sh``):
```
Comment (sverhoeven) [inline docs/hpc_to_cluster.rst]: Use editor that shows you how to get out
```suggestion
Here we briefly explain general SBATCH parameters and how to launch a Jupyter Lab environment on Cartesius. Start by opening a text editor on Cartesius (e.g. ``nano``) or (easier) your local machine (e.g. notepad). Copy the following text inside your text editor, edit the Conda environment name, and save as **run_jupyter_on_cartesius.sh** (make sure the extension is ``.sh``):
```

File: docs/hpc_to_cluster.rst [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+
+More information on job control can be found here: https://userinfo.surfsara.nl/systems/lisa/user-guide/creating-and-running-jobs#interacting
+
+=====================================
+Launching Jupyter Lab on Cluster Node
+=====================================
+
+**1. Open Slurm output log file**
+
+- Open slurm output log file by double clicking in the file browser or by using a text editor (``vim``) and read the output carefully.
```
Comment (sverhoeven) [inline docs/hpc_to_cluster.rst]:
```suggestion
- Open slurm output log file by double clicking in the file browser or by using a text editor (``nano``) and read the output carefully.
```

**Review by sverhoeven — APPROVED** [review]
Generated docs (https://ewatercycle--34.org.readthedocs.build/en/34/hpc_to_cluster.html) looks OK. 2 lines are bit long.

One major suggestion about grpc4bmi syntax and some minor suggestions. 
Could not help myself doing a review on the contents.

**Comment by SarahAlidoost** [comment]
> Generated docs (https://ewatercycle--34.org.readthedocs.build/en/34/hpc_to_cluster.html) looks OK. 2 lines are bit long.
> 
> One major suggestion about grpc4bmi syntax and some minor suggestions.
> Could not help myself doing a review on the contents.

Thank you for reviewing. The content might change in the future.

**Pull request merged**